src/k8s-engine/wsl.ts

```diff
@@ -1283,4 +1304,16 @@ export default class WSLBackend extends events.EventEmitter implements K8s.Kuber
     }
     console.log(`kubeconfig integration for ${ distro } set to ${ state }`);
   }
+
+  async getFailureDetails(): Promise<K8s.FailureDetails> {
+    const logfile = path.join(paths.logs, 'wsl.log');
```

Comment:
See above.

Reply:
Could you be more specific?

Reply:
https://github.com/rancher-sandbox/rancher-desktop/pull/1039#discussion_r760449993 — use `console.path` instead.

Reply:
Done